content/en/tracing/trace_collection/library_config/_index.md

```diff
@@ -40,6 +40,49 @@ The following configuration options behave consistently across the latest versio
 **Supported Input**: Boolean <br>
 **Description**: Enables or disables sending traces from the application.
 
+`DD_TRACE_SAMPLE_RATE`
+: **Default**: `-1`. If unset, the tracer defers to the Datadog Agent to control sample rate. <br>
+**Supported Input**: A float between 0.0 and 1.0, inclusive.  <br>
+**Caveats**: This variable is deprecated in favor of `DD_TRACE_SAMPLING_RULES`, which provides more flexible and granular sampling control.  <br>
+**Description**: Controls the trace ingestion sample rate between the Datadog Agent and the backend. Must be a float between 0.0 and 1.0, where 1.0 means all traces are sent to the backend and 0.0 means none are sent. This setting applies globally to all traces and does not support per-service or per-operation targeting.
+
+`DD_TRACE_SAMPLING_RULES`
+: **Default**: `null`. If unset or no rules match, the tracer defers to the Datadog Agent to dynamically adjust sample rate across traces.  <br>
+**Supported Input**: A JSON array of objects. <br>
+**Caveats**: Matching by `resource` and `tags` is in Preview.  <br>
+**Description**: Enables fine-grained control over trace ingestion, allowing you to target specific services, operations, resources, or tagged traces. Defined by a JSON array of objects, where each object must include a `sample_rate` between 0.0 and 1.0 (inclusive), and can optionally include fields such as `service`, `name`, `resource`, `tags`, and `max_per_second`. Objects are evaluated in the order listed; the first matching object determines the trace's sample rate. For more information, see [Ingestion Mechanisms][4]. <br>
+**Examples**: <br>
```

Comment:
```suggestion
**Use cases**: <br>
```

@duncanhewett wdyt about this verbiage?

Reply:
I think examples is fine personally, but happy either way!